processing/data_collection/gazette/spiders/rj_campos_goytacazes.py

```diff
@@ -0,0 +1,67 @@
+import re
+import dateparser
+
+from datetime import datetime
+from scrapy import Request
+from scrapy.exceptions import CloseSpider
+from gazette.items import Gazette
+from gazette.spiders.base import BaseGazetteSpider
+
+
+class RjCampoGoytacazesSpider(BaseGazetteSpider):
+    GAZETTE_ELEMENT_CSS = "ul.ul-licitacoes li"
+    TERRITORY_ID = "3301009"
+
+    allowed_domains = ["www.campos.rj.gov.br"]
+    name = "rj_campos_goytacazes"
+    start_urls = [
+        "https://www.campos.rj.gov.br/diario-oficial.php?PGpagina=1&PGporPagina=15"
+    ]
+    LIMIT_DATE = dateparser.parse("2015-01-01").date()
+
+    def parse(self, response):
+        """
+        @url https://www.campos.rj.gov.br/diario-oficial.php?PGpagina=1&PGporPagina=15
+        @returns requests 1
+        @returns items 15 15
+        @scrapes date file_urls is_extra_edition municipality_id power scraped_at
+        """
+
+        for element in response.css(self.GAZETTE_ELEMENT_CSS):
+            gazette_text = element.css("h4::text").extract_first() or ""
```

Comment:
**suggestion** Scrapy suggest the use of `.get` instead of (deprecated) `extract_first`. 

**suggestion** `get` (and `extract_first`) accepts a `default` [argument](https://parsel.readthedocs.io/en/latest/parsel.html#parsel.selector.SelectorList.get), so you don't need to add this `or ""`

This is equivalent to:
`gazette_text = element.css("h4::text").get("")`